specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
`cvr` now contains `ballot_manifest_info`.

When you say "index .. by", what do you mean in the modeling context?  Do you mean that an order can be defined on the non-indexed elements, or only that there is an injection from the index to the remaining projections?